de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/details/DetailsViewPresenterImpl.java

```diff
@@ -78,7 +270,67 @@ public DetailsView getView() {
         return view;
     }
 
-    List<Tag> wrapInList(Tag tag) {
-        return Lists.newArrayList(tag);
+    @Override
+    public void onTagSelection(String tagId, String tagValue) {
+        Tag tag = factory.getTag().as();
```

Comment:
Copying from #289:

Just a suggestion, but not required for this PR: This method could accept a `Splittable` tag that can be encoded here instead.